index.js

```diff
@@ -6,9 +6,70 @@ var babel = require('babel-core');
 var VersionChecker = require('ember-cli-version-checker');
 var Funnel = require('broccoli-funnel');
 var mergeTrees = require('broccoli-merge-trees');
+var fastbootTransform = require('fastboot-transform');
+var esTranspiler = require('broccoli-babel-transpiler');
 
 module.exports = {
   name: 'ember-cli-foundation-6-sass',
+
+  jsFilesToInclude: [],
+
+  treeForVendor(vendorTree) {
+    var foundationTree;
+    var options = this.app.options['ember-cli-foundation-6-sass'];
+    var checker = new VersionChecker(this);
+    var isGTE6_3_0 = checker.for('foundation-sites', 'npm').satisfies('>=6.3.0');
+
+    // Calculate the path using require.resolve which checks the whole path.
+    // This gives us a specific file in dist/js/npm.js hence the need to path.resolve our way back up.
+    var foundationPath = path.resolve(require.resolve('foundation-sites'), '../..');
+
+    if (options && options.foundationJs) {
+      if ((typeof options.foundationJs === 'string') ||
+        (options.foundationJs instanceof String)) {
+        if (options.foundationJs === 'all') {
+          // >=6.3.0 changed some paths.
+          if (isGTE6_3_0) {
+            foundationPath = path.resolve(require.resolve('foundation-sites'), '../../../dist/js'); // go deeper
+          }
+
+          foundationTree = new Funnel(foundationPath, {
+            destDir: 'foundation-sites',
+            files: this.jsFilesToInclude
+          });
+        }
+      } else {
+        if (isGTE6_3_0) {
+          foundationPath = path.resolve(require.resolve('foundation-sites'), '../../../js'); // go deeper
+        }
+
+        foundationTree = new Funnel(foundationPath, {
+          destDir: 'foundation-sites',
+          files: this.jsFilesToInclude
+        });
+
+        foundationTree = esTranspiler(foundationTree, {
+          plugins: [
+            require.resolve('babel-plugin-transform-es2015-arrow-functions'),
```

Comment:
Uh oh, I removed these babel transform dependencies in PR #90.

I'm a little surprised the `esTranspiler` is working.